features/bootstrap/FeatureContext.php

```diff
@@ -1,7 +1,41 @@
 <?php
 
-include_once(__DIR__ . '/../../php/class-ee.php');
-include_once(__DIR__ . '/../../php/utils.php');
+define('EE_ROOT',__DIR__ . '/../..');
```

Comment:
Add a space as you're doing it other places, after brackets.
ref: https://github.com/EasyEngine/easyengine/pull/1140/files#diff-8a50a8f3567ab9a03107fd86eeaf827eR10